event-svc/src/store/sql/entities/event.rs

```diff
@@ -45,20 +45,26 @@ pub async fn rebuild_car(blocks: Vec<BlockRow>) -> Result<Option<Vec<u8>>> {
 pub struct EventInsertable {
     /// The event order key (e.g. EventID)
     order_key: EventId,
+    /// The CID of init event of the stream
+    stream_cid: Cid,
     /// The event CID i.e. the root CID from the car file
     cid: Cid,
     /// Whether the event is deliverable i.e. it's prev has been delivered and the chain is continuous to an init event
     deliverable: bool,
     /// The parsed structure containing the actual Event data.
     event: Arc<unvalidated::Event<Ipld>>,
+    /// The DID of the node from which this event was received
+    source: Option<String>,
 }
 
 impl EventInsertable {
     /// EventInsertable constructor
     pub fn new(
         order_key: EventId,
+        stream_cid: Cid,
         event_cid: Cid,
         event: unvalidated::Event<Ipld>,
+        source: Option<String>,
```

Comment:
I see we add stream_cid, source, *and* anchored to the sql table, but we're only adding the first two here.  Does this need to take `anchored` also?

Reply:
We just took out `anchored`. We were planning to use it in part 5 but are not doing so anymore.

Reply:
in that case do we still need it in the sql table?

Reply:
I think [it was](https://github.com/ceramicnetwork/rust-ceramic/pull/484/files#diff-98a5d0fd33ad9eeaeafa6bf8b5092b30f73a3d5d2b63f752fba599fbaa90c8d6) removed, unless "it" is not `anchored` 😅.. `init_cid` and `informant` were added